### ISSUE TITLE
Items not found in Gemini should be reported as Warnings or Failed Events

### ISSUE THREAD

**Issue by seth-shaw-unlv** [mjordan/riprap#45]
_TL;DR: If Gemini reports back a 404 we should at least issue a warning to the user._

While testing riprap I came across an issue where the [uuid for files in Drupal did not match the uuid recorded in Gemini](https://github.com/Islandora-CLAW/CLAW/issues/1076). However, when riprap attempted to check 49 items, the status message was misleading:
```
Riprap checked 5 resources (5 successful events, 0 failed events) in 18.195 seconds.
```
I added some print lines to PluginFetchResourceListFromDrupal.php for more information which reported:
```
# php bin/console app:riprap:check_fixity --settings=local-config.yaml 
MISSING: fedora uri for file 'ent001201-007.tif' with uuid b89f822c-34de-4394-bfda-bea5e95d328b
MISSING: fedora uri for file 'ent001330-162.tif' with uuid 500a6720-177c-4b00-859f-00ee5b9bd3e6
MISSING: fedora uri for file 'ent001405-084.tif' with uuid c030366f-6de0-4377-afbe-6f9afbcc4c4d
MISSING: fedora uri for file 'ent001459-016.tif' with uuid 5c1cb952-7755-4921-b6ac-84a784d1393a
MISSING: fedora uri for file 'ent001539-015.tif' with uuid 0d9d4930-e0ea-470a-8b72-983f3c0bc56f
FOUND: fedora uri for file 'ent001671-011.tif' with uuid c05840d2-0b50-4109-99c1-ad52dfdb7bad
FOUND: fedora uri for file 'ent000301-015.tif' with uuid d1ca1e4e-2cc9-4304-aeb7-c2acfd9eb8fd
MISSING: fedora uri for file 'ent000509-035.tif' with uuid ee19d7e1-4f09-4562-ba44-02b97a94d6d6
MISSING: fedora uri for file 'ent000945-114.tif' with uuid d5e12d82-d09e-4194-b885-33002904970a
MISSING: fedora uri for file 'ent000991-267.tif' with uuid 192c2c6a-fee7-471c-bd4d-0e298aef4703
MISSING: fedora uri for file 'ent001075-182.tif' with uuid 95183bc7-e80d-446d-9b76-6ede9bc2c36f
MISSING: fedora uri for file 'ent001201-023.tif' with uuid 37392de4-ef22-4cb3-881c-2d1ab40bf93f
MISSING: fedora uri for file 'ent001330-178.tif' with uuid 84825de8-122c-4646-8a50-22ce01b14b5c
MISSING: fedora uri for file 'ent001405-100.tif' with uuid 9134f6ee-d23c-4ab4-aafd-15eb2ab55cc3
MISSING: fedora uri for file 'ent001459-032.tif' with uuid 0779fa35-e2b9-4960-a1f9-06a77bd03c6e
MISSING: fedora uri for file 'ent001539-031.tif' with uuid 73605efc-10e1-4434-93b3-2dc9eb454663
FOUND: fedora uri for file 'ent001671-027.tif' with uuid 7a41ba6b-9aea-4b47-ae12-4db0112c76a9
FOUND: fedora uri for file 'ent000301-031.tif' with uuid 2ff93317-cea9-4ae8-b45b-fa6356628469
MISSING: fedora uri for file 'ent000509-051.tif' with uuid 7a70fafd-1941-4ccb-a685-2fe6f9253f3a
MISSING: fedora uri for file 'ent000946-012.tif' with uuid d90120d2-33cf-4592-b6ce-01e503c80d32
MISSING: fedora uri for file 'ent000991-283.tif' with uuid ffd51f68-dc91-4da9-b2ca-29ed61e8be3c
MISSING: fedora uri for file 'ent001075-198.tif' with uuid 9b08b139-22d1-4c1b-9965-6cc6e24ef308
MISSING: fedora uri for file 'ent001202-011.tif' with uuid e90a1303-601e-4e11-8fac-2b223801642d
MISSING: fedora uri for file 'ent001330-194.tif' with uuid fee65a73-c155-44cb-972e-073f8981627e
MISSING: fedora uri for file 'ent001405-116.tif' with uuid aa7b9f0b-49f8-49e1-8df4-82baef60a27d
MISSING: fedora uri for file 'ent001459-048.tif' with uuid 557d6258-dd37-46a5-b3c1-6d99d06daa03
FOUND: fedora uri for file 'ent001671-043.tif' with uuid 0bd4f921-05d8-4e49-86ed-65e17c884bd0
FOUND: fedora uri for file 'ent000302-007.tif' with uuid eb7db427-cd8f-48d3-a4be-79ec4b9876e4
MISSING: fedora uri for file 'ent000509-067.tif' with uuid afe2eca5-9c31-42c4-b4db-6c632fe5ae64
MISSING: fedora uri for file 'ent000778-002.tif' with uuid d1ffd9da-7a90-4461-9bc4-a442e88e743c
MISSING: fedora uri for file 'ent000946-028.tif' with uuid a2e91cdf-bf04-49dc-a297-a0241d0b3b1f
MISSING: fedora uri for file 'ent000991-299.tif' with uuid b44f6219-7e0e-45ca-a74a-087cb1475532
MISSING: fedora uri for file 'ent001075-214.tif' with uuid c77db041-978e-4707-9dfe-dc62bb154759
MISSING: fedora uri for file 'ent001202-027.tif' with uuid 29598ef8-e06e-41f6-ada2-53387dd5db67
MISSING: fedora uri for file 'ent001330-210.tif' with uuid 6e3ffa56-51ef-4072-95bb-ff5c4b76f96a
MISSING: fedora uri for file 'ent001405-132.tif' with uuid 36c1f2f8-1f73-4470-a790-eaee819ce72a
MISSING: fedora uri for file 'ent001460-009.tif' with uuid 0608f148-d397-4b6b-898c-fd5cd6482d85
FOUND: fedora uri for file 'ent001672-003.tif' with uuid 4b61d0f5-7c86-4100-bf74-20d887fdbfd8
MISSING: fedora uri for file 'ent000509-083.tif' with uuid 396febbb-15c0-4310-84e9-aa744ab76b6f
MISSING: fedora uri for file 'ent000780-005.tif' with uuid 583020da-a905-4112-8034-2390bbb7cb8f
MISSING: fedora uri for file 'ent000946-044.tif' with uuid 21eff1b8-429b-46db-b3a5-acf76a1b9e06
MISSING: fedora uri for file 'ent000991-315.tif' with uuid fd92a0e2-6769-4c05-a72e-49f2798ece43
MISSING: fedora uri for file 'ent001075-230.tif' with uuid 4f1ee155-3563-419f-a6de-6eb8377a87e1
MISSING: fedora uri for file 'ent001203-001.tif' with uuid 3c283ad4-488e-45a9-a308-93634cdc04e2
MISSING: fedora uri for file 'ent001330-226.tif' with uuid 2affe354-7dcf-4e8e-a065-fe6fd30d9fd4
Riprap checked 7 resources (7 successful events, 0 failed events) in 17.627 seconds.
```

All of these 'MISSING' errors can be explained by the Gemini uuid mismatch issue, but other users may incorrectly assume that riprap checked ONLY 7 items despite their configuration being set to 50. If Gemini reports back a 404, we should at least issue a warning to the user.

**Comment by whikloj** [comment]
Gemini is no longer a thing, so this could be closed as (I'm assuming) a 404 from Fedora would result in a warning.
